lib/tests/streamlit/runtime/caching/hashing_test.py

```diff
@@ -211,6 +212,22 @@ def test_numpy(self):
 
         self.assertEqual(get_hash(np4), get_hash(np5))
 
+    def test_PIL_image(self):
+        im1 = Image.new("RGB", (50, 50), (220, 20, 60))
+        im2 = Image.new("RGB", (50, 50), (30, 144, 255))
+        im3 = Image.new("RGB", (50, 50), (220, 20, 60))
+
+        self.assertEqual(get_hash(im1), get_hash(im3))
+        self.assertNotEqual(get_hash(im1), get_hash(im2))
+
+        # Check for big PIL images, they converted to numpy array with size
+        # bigger than _NP_SIZE_LARGE
+        # 1000 * 1000 * 3 = 3_000_000 > _NP_SIZE_LARGE = 1_000_000
+        im4 = Image.new("RGB", (1000, 1000), (100, 20, 60))
+        im5 = Image.new("RGB", (1000, 1000), (100, 20, 60))
+
+        self.assertEqual(get_hash(im4), get_hash(im5))
```

Comment:
Can you add another `self.assertNotEqual` test down here for big images, similar to the one for small images above?